src/caselawclient/models/documents.py

```diff
@@ -278,6 +284,10 @@ def transformation_datetime(self) -> Optional[datetime.datetime]:
     def enrichment_datetime(self) -> Optional[datetime.datetime]:
         return self.get_latest_manifestation_datetime("tna-enriched")
 
+    @cached_property
+    def reparse_datetime(self) -> Optional[datetime.datetime]:
+        return self.get_latest_manifestation_datetime("last_sent_to_parser")
```

Comment:
This can't work, can it? This needs to be a MarkLogic property I think, rather than getting the thing from the XML document.

Reply:
The enrichment one can't work either -- it has to be looking at the properties!